storage/integration_test.go

```diff
@@ -4378,34 +4396,77 @@ func TestIntegration_PostPolicyV4(t *testing.T) {
 
 // Verify that custom scopes passed in by the user are applied correctly.
 func TestIntegration_Scopes(t *testing.T) {
-	// A default client should be able to write objects since it has scope of
-	// FullControl
-	ctx := context.Background()
-	clientFullControl := testConfig(ctx, t)
-	defer clientFullControl.Close()
+	multiTransportTest(context.Background(), t, func(t *testing.T, ctx context.Context, bucket, _ string, client *Client) {
```

Comment:
What do we think about having multiple `multiTransportTest` calls in a single integration test? Maybe this should be broken up? On the other hand if it doesn't break anything, probably fine... @noahdietz